nncf/quantization/structs.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, quantizer_module_ref: BaseQuantizer,
 class WeightQuantizerInfo:
     def __init__(self,
                  quantizer_module_ref: BaseQuantizer,
-                 quantized_module: torch.nn.Module):
+                 quantized_module: torch.nn.Module,
+                 affected_insertions: List[PTTargetPoint]):
         self.quantizer_module_ref = quantizer_module_ref
         self.quantized_module = quantized_module
+        self.affected_insertions = affected_insertions
```

Comment:
please remove, if it's not used anywhere

Reply:
It is necessary to know which points in the graph a given quantizer module quantizes. A single quantizer module may apply its quantization to different points in the model's graph. This is the mechanism behind unified scales.

Reply:
in that case it makes sense to define interface or base class (e.g. 'QuantizerInfo') to access common fields

Reply:
Done